setup.py

```diff
@@ -8,7 +8,8 @@
     url='https://github.com/parrt/dtreeviz',
     license='MIT',
     packages=find_packages(),
-    install_requires=['graphviz>=0.9','pandas','numpy','scikit-learn','matplotlib','colour', 'xgboost', 'pytest', 'pyspark'],
+    install_requires=['graphviz>=0.9','pandas','numpy','scikit-learn','matplotlib','colour', 'xgboost', 'pytest', 'pyspark',
```

Comment:
during your cleanup, don't forget to re-base to pull in the dynamic loading of libraries :) I think we merged a PR for that, but I could be confusing it with one of the other billion Python libraries I'm trying to track in my head right now. haha!

Reply:
Sure, I've already merged upstream master with my local lightgbm branch

Reply:
Ah right. ok, i'll hold off on this PR until you've updated. thanks!